src/components/ReactTags.js

```diff
@@ -440,6 +443,7 @@ class ReactTags extends Component {
           id={inputId}
           maxLength={maxLength}
           value={this.props.inputValue}
+          disabled={disabled}
```

Comment:
as discussed in previous Pull request, these props will not be needed if we pass inputProps as rest props